R/utils.R

```diff
@@ -237,3 +237,11 @@ fileMD5.as.string <- function(path) {
 md5.as.string <- function(md5) {
   paste(md5, collapse = "")
 }
+
+isAvailable <- function(package, version = NULL) {
+  installed <- nzchar(system.file(package = package))
+  if (is.null(version)) {
+    return(installed)
+  }
+  installed && isTRUE(utils::packageVersion(package) >= version)
```

Comment:
This comparison is incorrect, as we want version comparison via `compareVersion`, not string comparisons.

Addressing myself, as we are actively testing.

Reply:
Wrong about this; local testing was relying on strings when testing version comparisons, not the object returned by `packageVersion`.